critiquebrainz/frontend/external/bookbrainz_db/identifiers.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
+from brainzutils import cache
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+
+def fetch_identifiers(identifier_set_id: int) -> List:
```

Comment:
can we have a name that's more descriptive than `identifiers`? maybe they're bookbrainz_external_site_identifiers?

Reply:
@MonkeyDo do you have a better idea?